[2024-08-02T13:50:47.452+0700] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-08-02T13:50:47.469+0700] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: data_pipeline_supermarket.airbyte_check_sync_1 manual__2024-08-02T03:30:51.725028+00:00 [queued]>
[2024-08-02T13:50:47.484+0700] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: data_pipeline_supermarket.airbyte_check_sync_1 manual__2024-08-02T03:30:51.725028+00:00 [queued]>
[2024-08-02T13:50:47.484+0700] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-08-02T13:50:47.508+0700] {taskinstance.py:2330} INFO - Executing <Task(AirbyteJobSensor): airbyte_check_sync_1> on 2024-08-02 03:30:51.725028+00:00
[2024-08-02T13:50:47.512+0700] {standard_task_runner.py:64} INFO - Started process 5811 to run task
[2024-08-02T13:50:47.514+0700] {standard_task_runner.py:90} INFO - Running: ['airflow', 'tasks', 'run', 'data_pipeline_supermarket', 'airbyte_check_sync_1', 'manual__2024-08-02T03:30:51.725028+00:00', '--job-id', '94', '--raw', '--subdir', 'DAGS_FOLDER/data_pipeline_supermarket.py', '--cfg-path', '/tmp/tmp228ck40k']
[2024-08-02T13:50:47.516+0700] {standard_task_runner.py:91} INFO - Job 94: Subtask airbyte_check_sync_1
[2024-08-02T13:50:47.557+0700] {task_command.py:426} INFO - Running <TaskInstance: data_pipeline_supermarket.airbyte_check_sync_1 manual__2024-08-02T03:30:51.725028+00:00 [running]> on host DESKTOP-RHVHV7A.
[2024-08-02T13:50:47.657+0700] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='data_pipeline_supermarket' AIRFLOW_CTX_TASK_ID='airbyte_check_sync_1' AIRFLOW_CTX_EXECUTION_DATE='2024-08-02T03:30:51.725028+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-08-02T03:30:51.725028+00:00'
[2024-08-02T13:50:47.658+0700] {taskinstance.py:430} INFO - ::endgroup::
[2024-08-02T13:50:47.681+0700] {baseoperator.py:400} WARNING - AirbyteJobSensor.execute cannot be called outside TaskInstance!
[2024-08-02T13:50:47.687+0700] {base.py:84} INFO - Using connection ID 'airflow-call-to-airbyte-example' for task execution.
[2024-08-02T13:50:47.736+0700] {airbyte.py:108} INFO - Job 70 completed successfully.
[2024-08-02T13:50:47.736+0700] {base.py:294} INFO - Success criteria met. Exiting.
[2024-08-02T13:50:47.737+0700] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-08-02T13:50:47.746+0700] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=data_pipeline_supermarket, task_id=airbyte_check_sync_1, run_id=manual__2024-08-02T03:30:51.725028+00:00, execution_date=20240802T033051, start_date=20240802T065047, end_date=20240802T065047
[2024-08-02T13:50:47.768+0700] {local_task_job_runner.py:243} INFO - Task exited with return code 0
[2024-08-02T13:50:47.778+0700] {logging_mixin.py:188} WARNING - /usr/local/lib/python3.10/dist-packages/airflow/models/baseoperator.py:1297 AirflowProviderDeprecationWarning: Call to deprecated class PostgresOperator. (Please use `airflow.providers.common.sql.operators.sql.SQLExecuteQueryOperator`.Also, you can provide `hook_params={'schema': <database>}`.)
[2024-08-02T13:50:47.799+0700] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-08-02T13:50:47.799+0700] {local_task_job_runner.py:222} INFO - ::endgroup::
